core/src/main/java/org/apache/calcite/runtime/SqlFunctions.java

```diff
@@ -462,6 +462,16 @@ public static ByteString right(ByteString s, int n) {
     return s.substring(len - n);
   }
 
+  /**
+   * SQL CHAR(int) function.
+   */
```

Comment:
explain the difference between this and `chr`. It isn't int vs long.

Reply:
also make the javadoc consistent with other javadoc

Reply:
Yes, it should be bigint.